lib/matplotlib/cbook.py

```diff
@@ -2697,40 +2715,49 @@ class _StringFuncParser(object):
     _funcs['linear'] = _FuncInfo(lambda x: x,
                                  lambda x: x,
                                  True)
-    _funcs['quadratic'] = _FuncInfo(lambda x: x**2,
-                                    lambda x: x**(1. / 2),
+    _funcs['quadratic'] = _FuncInfo(np.square,
+                                    np.sqrt,
                                     True)
     _funcs['cubic'] = _FuncInfo(lambda x: x**3,
```

Comment:
But I am not sure how would that help, you would still need the lambda function to pass the kwargs, right? 

In any case, I do not think it really is that relevant at this point, we are trying to do something relatively simple for a subset of functions (for which it can be tested) aimed to help FuncNorm and children. If we ever decide to make this accessible class directly accesible to end users, or to let users create their own cuestom _FuncInfo objects, then we can worry about these things.